smartmin/models.py

```diff
@@ -303,6 +325,7 @@ def unicode_csv_reader(utf8_data, dialect=csv.excel, **kwargs):
         if import_results is not None:
             import_results['records'] = len(records)
             import_results['errors'] = num_errors
+            import_results['error_description'] = error_description
```

Comment:
Ya, no i was thinking like compile errors you just show first 10 or 20 or so.. then say how many total there were.